project_parameters.Config.yaml

```diff
@@ -134,6 +133,10 @@ celltype_reference_module: "predicted.cell.signature.ident" # Column name from t
 annotations_dir_rshiny_app: "cell_types_annotations_all" # Options: to update based on the folder with the desired annotations to use for the R shiny app from the `cell-types-annotation` module
 annotations_filename_rshiny_app: "seurat_obj_cell_types_annotations_all.rds" # Options: to update based on the object with the desired annotations to use for the R shiny app from the `cell-types-annotation` module
 
+# `rshiny-app`
+annotations_dir_rshiny_app: "cell_types_annotations_all" # Options: to update based on the folder with the desired annotations to use for the R shiny app from the `cell-types-annotation` module
+annotations_filename_rshiny_app: "seurat_obj_cell_types_annotations_all.rds" # Options: to update based on the object with the desired annotations to use for the R shiny app from the `cell-types-annotation` module
+
```

Comment:
`r-shiny-app` parameters are already defined above, so these are duplicates and will cause an error when reading the `yaml` file in for scripts.

```suggestion
```